bip-0174.mediawiki

```diff
@@ -133,8 +133,8 @@ The currently defined global types are as follows:
 *** <tt>{32-bit uint}</tt>
 
 * Type: Proprietary Use Type <tt>PSBT_GLOBAL_PROPRIETARY = 0xFC</tt>
-** Key: Variable length identifier prefix, followed by a subtype, followed by the key data itself.
-*** <tt>{0xFC}|<prefix>|{subtype}|{key data}</tt>
+** Key: Compact size unsigned integer, followed by identifier prefix of that length, followed by a subtype, followed by the key data itself.
+*** <tt>{0xFC}|{prefixlen}|<prefix>|{subtype}|{key data}</tt>
```

Comment:
The note on notation found earlier in the BIP indicates that `<...>` means "prefixed by compact size unsigned integer. So this is technically incorrect.

Reply:
Thanks, I missed that!

Is this boutique notation used in other BIPs?  I haven't seen it before and it was completely non-intuitive to me!

Reply:
> Is this boutique notation used in other BIPs? I haven't seen it before and it was completely non-intuitive to me!

I think I made up the notation as a placeholder and just never changed it to something else.